ReactS3Uploader.js

```diff
@@ -53,7 +54,8 @@ var ReactS3Uploader = createReactClass({
             scrubFilename: function(filename) {
                 return filename.replace(/[^\w\d_\-\.]+/ig, '');
             },
-            s3path: ''
+            s3path: '',
+            inputRef: function(el){}
```

Comment:
Probably don't need this, since you can just pass an `undefined` as the `ref`

Reply:
Sure thing :)